test/integration/test-transformer.ts

```diff
@@ -40,6 +40,7 @@ export default {
 						'hS6LAf/gt343P/gPihbshbd1zdEJGqYwcmyooK3HuwcFqgjajBiwX322YwI+WDlJP7ilUzCAspbWx4/Ch4YuRw==',
 				},
 			},
+			expectedOutputTypes: ['transformer', 'docker-image'],
```

Comment:
These are just test values, but not sure if the actual ones should have versions...

Reply:
yes, I think the actual ones should have versions. At least switching major versions would get hairy otherwise

Reply:
Cool, added versions now.